### DISCUSSION TITLE
Where are the software grabbers located?

### DISCUSSION THREAD

**Discussion by chcore** [awawa-dev/HyperHDR#296] [Q&A]
I have downloaded the latest beta release for x86_64 linux, and also successfully compiled HyperHDR, but I don't see the software grabber executables anywhere. Where should I be looking?

**Comment by awawa-dev** [comment]
Never existed as standalone applications in HyperHDR

**Comment by chcore** [comment]
How would I go about using them?

Currently I have HyperHDR running on my rPI (which controls my APA102 LEDs), and I want to capture the video signal from a different linux box (e.g. I would use hyperion-xcb or something similar if I were using hyperion). Is this not possible?

**Comment by awawa-dev** [comment]
You have two options:
- using flatbuffer forwarder (advance->network, you may need to enable advanced user settings to have it visible there). It's explained here https://github.com/awawa-dev/HyperHDR/discussions/149 . Since it's sending raw video frames from one HyperHDR instance to another, it can easily saturate the network for high framerate & large frames causing lag.
-  use new v18 feature Advanced → Network services → UDP raw receiver ( https://postimg.cc/239dK79j ) to receive RGB values from other HyperHDR instance. In fact it turns HyperHDR into 'WLED'. It has very small impact on the network. You must set the same LED strip geometry for both HyperHDR instances and disable smoothing on the sender (otherwise smoothing will be applied twice)

**Comment by chcore** [comment]
Thank you, I will try this and see how goes.

Is there any merit to me raising an issue for allowing or providing standalone grabbers with HyperHDR?

**Comment by awawa-dev** [comment]
I dont plan standalone grabbers.
